.github/workflows/ci.yml

```diff
@@ -65,9 +65,9 @@ jobs:
 
       - name: Test Docker Container listing
         run: docker ps
-      
-      - name: API healthcheck
-        run: curl -sSLf --retry-delay 5 --retry 5 --retry-connrefused --insecure http://example.org > /dev/null
```

Comment:
The problem here was the wrong url `http://example.org`, right? Did you try this with `localhost:8080/actuator/health`?

Reply:
Sure, but long story short: It did not work either. I'd suggest to merge this as a temporary solution.

Reply:
ok